### ISSUE TITLE
Bug in explain_prediction_xgboost?

### ISSUE THREAD

**Issue by louis-vines** [TeamHG-Memex/eli5#288]
Hi All

I really like your package - find it super useful. However I think I've found a bug in the `explain_prediction_xgboost` function.

Here is my example:

```
>>> explain_prediction_xgboost(model, X_test.values[0], is_regression=True)

--------------------------------------------------------------
TypeError                    Traceback (most recent call last)
<ipython-input-78-95e8c14d9241> in <module>()
----> 1 explain_prediction_xgboost(model, X_test.values[0], is_regression=True)

~/miniconda3/envs/science/lib/python3.6/site-packages/eli5/xgboost.py in explain_prediction_xgboost(xgb, doc, vec, top, top_targets, target_names, targets, feature_names, feature_re, feature_filter, vectorized, is_regression, missing)
    146     Weights of all features sum to the output score of the estimator.
    147     """
--> 148     booster, is_regression = _check_booster_args(xgb, is_regression)
    149     xgb_feature_names = booster.feature_names
    150     vec, feature_names = handle_vec(

~/miniconda3/envs/science/lib/python3.6/site-packages/eli5/xgboost.py in _check_booster_args(xgb, is_regression)
    223         booster = xgb
    224     else:
--> 225         booster = xgb.booster()
    226         _is_regression = isinstance(xgb, XGBRegressor)
    227         if is_regression is not None and is_regression != _is_regression:

TypeError: 'str' object is not callable
```

I've found it change `eli5/xgboost.py::225` to:

```
booster = xgb.get_booster()
```
 the function works as I expect it to.

I don't know if maybe this is due to the API of xgboost changing or not.

Happy to put a PR in to fix this if needs be.

**Comment by lopuhin** [comment]
@louis-vines thanks for the bug report! I suspect this is fixed in master, but not released on PyPI yet, any chance you could try it out?

**Comment by louis-vines** [comment]
I checked the source code on master and you're right it has been fixed.

Thanks for the swift reply!

**Comment by lopuhin** [comment]
Thanks for confirmation! Closing (the fix is released on PyPI already too)
